### ISSUE TITLE
Provide column importance scores via lightwood

### ISSUE THREAD

**Issue by George3d6** [mindsdb/lightwood#70]
This should probably come after #68 and #69 

We could try providing some sort of column importance score from lightwood based on:

a) The results of training with drop-out (#68), i.e. if dropping out a given column yields a low accuracy then it's probably very important and vice-versa
b) The correlation that certain encoder find between the IR of the column and the target variable (see #69)
c) *Maybe* some analysis of the input weights corresponding to the inputs derived from the column and/or a more in-depth analysis of their importance in the graph and/or maybe even the gradient flow during training.

**Comment by torrmal** [comment]
* lets make the target 3 variables

  * target_mean
  * predicted_error
  * target_95pc_confidence_lower_band
  * target_95pc_confidence_higher_band

* quantile  loss functions for target_mean, target_95pc_confidence_lower_band, target_95pc_confidence_higher_band

```
class QuantileLoss(nn.Module):
    def __init__(self, quantiles):
        super().__init__()
        self.quantiles = quantiles
        
    def forward(self, preds, target):
        assert not target.requires_grad
        assert preds.size(0) == target.size(0)
        losses = []
        for i, q in enumerate(self.quantiles):
            errors = target - preds[:, i]
            losses.append(
                torch.max(
                   (q-1) * errors, 
                   q * errors
            ).unsqueeze(1))
        loss = torch.mean(
            torch.sum(torch.cat(losses, dim=1), dim=1))
        return loss
```
see https://medium.com/the-artificial-impostor/quantile-regression-part-2-6fdbc26b2629



* predicted_error loss function RMSE as we do now

make the total loss the sum, i think it should be fine its just a sum of all losses

With this, we can return a nice confidence interval that is calculated in a pro way

https://en.wikipedia.org/wiki/Confidence_interval

**Comment by George3d6** [comment]
This has been on the back-burner for so long that I'm closing it.

The quantile loss described by @torrmal above is implemented.
